docs/development/docs.rst

```diff
@@ -5,32 +5,77 @@ As one might expect,
 the documentation for Read the Docs is built using Sphinx and hosted on Read the Docs.
 The docs are kept in the ``docs/`` directory at the top of the source tree.
 
-.. TODO: expand this section explaining there the PR is automatically built and
-   the author can visualize changes without installing anything on their system.
-   However, if there is going to be periodic/bigger contributions, it may be a
-   good idea to install the Sphinx requirements to build our docs.
+Contributing through the Github UI
+----------------------------------
+
+If you're making small changes to the documentation,
+you can verify those changes through the documentation generated when you open a PR and can be accessed using the Github UI.
+
+#. click the checkmark next to your commit and it will expand to have multiple options
+
+#. right-click the "details" link next to the "docs/readthedocs.org:docs" item
+
+   .. image:: /img/details_link.png
+
+#. navigate to the section of the documentation you worked on to verify your changes
+
+Contributing from your local machine
+------------------------------------
+
+.. note::
+
+   You must use python 3.6 to generate the documentation.
+
+If you're making large changes to the documentation,
+you may want to verify those changes locally before pushing upstream.
+
+#. clone the `readthedocs.org` repository:
+
+   .. prompt:: bash
+
+      git clone --recurse-submodules https://github.com/readthedocs/readthedocs.org/
+
+#. install documentation requirements
+
+   .. prompt:: bash
+
+      cd readthedocs.org
+      pip install -r requirements/testing.txt
```

Comment:
Do we need `requirements/testing.txt` to work on the docs?

Reply:
At the moment they're needed, yes. Otherwise some dependencies are missing.

Reply:
(It's up to debate whether we should organize our dependencies in a better way, or improve the naming of the requirements files, see https://github.com/readthedocs/readthedocs.org/pull/7037#issuecomment-644960726)